centmin.sh

```diff
@@ -1438,20 +1427,14 @@ funct_centos6check
     export PHP_AUTOCONF=/usr/bin/autoconf
     export PHP_AUTOHEADER=/usr/bin/autoheader
 
-    cd "$DIR_TMP"
-
 if [ "$(rpm -qa | grep '^php*' | grep -v 'phonon-backend-gstreamer')" ]; then
   # IMPORTANT Erase any PHP installations first, otherwise conflicts may arise
   echo "${YUMDNFBIN} -y erase php*"
   ${YUMDNFBIN}${CACHESKIP} -y erase php*
 
 fi
 
-    #download php tarball
-    if [[ ! -d "php-${PHP_VERSION}" && -f "php-${PHP_VERSION}.tar.${PHPEXTSION}" ]]; then
-        tar xzvf "php-${PHP_VERSION}.tar.${PHPEXTSION}"
-    fi
```

Comment:
Unnecessary checking. phptarball() function already has a command to unpack PHP archive and also checks for errors..